TCAT/Controllers/RouteOptionsViewController.swift

```diff
@@ -402,51 +402,95 @@ class RouteOptionsViewController: UIViewController, UITableViewDelegate, UITable
     private func setupLocationManager() {
         locationManager = CLLocationManager()
         locationManager.delegate = self
-        locationManager.desiredAccuracy = kCLLocationAccuracyBest
+        locationManager.desiredAccuracy = kCLLocationAccuracyNearestTenMeters
         locationManager.distanceFilter = 10
-        locationManager.requestWhenInUseAuthorization()
+        locationManager.requestAlwaysAuthorization()
         locationManager.startUpdatingLocation()
     }
 
     func locationManager(_ manager: CLLocationManager, didFailWithError error: Swift.Error) {
         print("RouteOptionVC locationManager didFailWithError: \(error.localizedDescription)")
-        locationManager.stopUpdatingLocation()
-        var locationAuthReminder = true
         
-        if let showReminder = userDefaults.value(forKey: Constants.UserDefaults.locationAuthReminder) as? Bool {
-            if showReminder {
-                locationAuthReminder = false
+        if error._code == CLError.denied.rawValue {
```

Comment:
Should we make this a global function in Extensions?

Reply:
eh i think we only use it 2 places...let's keep it for now. But stmg I think we need to group up is the setting up the CLLocationManager. When I was doing this I realized we all set up our CLLocationManager slightly differently. I modified them so we all used the same attributes but we really should have 1 class or function that sets up the CLLocationManager uniformly